src/components/Video/Video.tsx

```diff
@@ -1,117 +1,61 @@
-import React, { useCallback, useEffect, useState } from 'react';
+import React from 'react';
 import classNames from 'classnames';
 import { useTranslation } from 'react-i18next';
 
 import styles from './Video.module.scss';
 
 import CollapsibleText from '#src/components/CollapsibleText/CollapsibleText';
 import Button from '#src/components/Button/Button';
-import IconButton from '#src/components/IconButton/IconButton';
-import Modal from '#src/components/Modal/Modal';
-import Fade from '#src/components/Animation/Fade/Fade';
-import Alert from '#src/components/Alert/Alert';
-import ModalCloseButton from '#src/components/ModalCloseButton/ModalCloseButton';
-import Cinema from '#src/containers/Cinema/Cinema';
 import useBreakpoint, { Breakpoint } from '#src/hooks/useBreakpoint';
 import Favorite from '#src/icons/Favorite';
 import PlayTrailer from '#src/icons/PlayTrailer';
-import Share from '#src/icons/Share';
-import Check from '#src/icons/Check';
-import ArrowLeft from '#src/icons/ArrowLeft';
-import { formatDuration } from '#src/utils/formatting';
 import FavoriteBorder from '#src/icons/FavoriteBorder';
-import type { PlaylistItem } from '#types/playlist';
-import { useFavoritesStore } from '#src/stores/FavoritesStore';
 
-type Poster = 'fading' | 'normal';
+type PosterMode = 'fading' | 'normal';
 
 type Props = {
   title: string;
-  item: PlaylistItem;
-  feedId?: string;
-  trailerItem?: PlaylistItem;
-  play: boolean;
-  goBack: () => void;
-  onComplete?: () => void;
-  isFavorited: boolean;
+  description: string;
+  videoMeta: string;
+  seriesMeta?: string;
+  episodeTitle?: string;
+  isFavorite: boolean;
   isFavoritesEnabled: boolean;
   onFavoriteButtonClick: () => void;
-  poster: Poster;
-  enableSharing: boolean;
-  hasShared: boolean;
-  onShareClick: () => void;
-  playTrailer: boolean;
+  poster?: string;
+  posterMode: PosterMode;
+  hasTrailer: boolean;
   onTrailerClick: () => void;
   onTrailerClose: () => void;
+  playTrailer: boolean;
   isSeries?: boolean;
-  episodeCount?: number;
   startWatchingButton: JSX.Element;
-  children?: JSX.Element;
+  shareButton: JSX.Element | null;
 };
 
 const Video: React.FC<Props> = ({
   title,
-  item,
-  feedId,
-  trailerItem,
-  play,
-  goBack,
-  onComplete,
+  description,
+  videoMeta,
+  seriesMeta,
+  episodeTitle,
   poster,
-  enableSharing,
-  hasShared,
-  onShareClick,
-  isFavorited,
+  posterMode,
+  isFavorite,
   isFavoritesEnabled,
```

Comment:
Suggest we pull out Favorites button the same as the Share and Start Watching buttons

Reply:
Also done!